docker/hadoop/Dockerfile

```diff
@@ -0,0 +1,71 @@
+#FROM debian:bullseye-slim
+FROM debian:bullseye
+
+# ssh service
+
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt update && apt install -y ssh #&& rm -rf /var/lib/apt/lists/*
+RUN mkdir /run/sshd
+
+# passwordless ssh localhost
+
+RUN mkdir /root/.ssh
+RUN ssh-keygen -t rsa -f /root/.ssh/id_rsa -N ''
+RUN cat /root/.ssh/id_rsa.pub >> /root/.ssh/authorized_keys
+RUN chmod og-wx /root/.ssh/authorized_keys
+
+# get resources
+
+RUN mkdir /root/Downloads
+WORKDIR /root/Downloads
+
+COPY ./files/pkg/hadoop-3.3.3.tar.gz                 /root/Downloads/.
```

Comment:
Esta lítea me da problema, ¿Es posible que la ruta relativa esté incorrecta? Aunque si intento hacer `../files/pkg/hadoop-3.3.3.tar.gz` me aparece el error: **forbidden path outside the build context**.

Reply:
Si, debe ser que no descargó el archivo.

Reply:
Correcto, solucionado con el último commit.